google/ggrc-core
# GGRC-8847 Due Date value is not saving in Create Assessments modal

# Issue description

When assessment was created with filled Due Date this data has been missed.

# Steps to test the changes

1. Log in GGRC app.
2. Open Create Assessment modal.
3. Fill Due Date field.
4. Click Save the assessment.
5. Expected: Due Date is successfully set after saving.

# Solution description

Changed data binding for datepicker-component on modal-content-additional-fields component

# Sanity checklist




# PR Review checklist
